content/administration/maintain/mailjet_api.rst

```diff
@@ -0,0 +1,221 @@
+============================
+Set up Mailjet API with Odoo
+============================
+
+Odoo is compatible with Mailjet's :abbr:`API (Application Programming Interface)` for mass mailing.
+Set up a dedicated mass mailing server through Mailjet by configuring settings in the Mailjet
+account and the Odoo database. In some circumstances, settings need to be configured on the custom
+domain's :abbr:`DNS (Domain Name System)` settings as well.
+
+Set up in Mailjet
+=================
+
+Create API credentials
+----------------------
+
+To get started, sign in to the `Mailjet Account Information <https://app.mailjet.com/account>`_
+page. Next, navigate to the :guilabel:`Senders & Domains` section and click on :guilabel:`SMTP and
+SEND API Settings`.
+
+.. image:: mailjet_api/api-settings.png
+   :align: center
+   :alt: SMTP and Send API Settings link in the Senders & Domains section of Mailjet.
+
+Then, copy the :abbr:`SMTP (Simple Mail Transfer Protocol)` configuration settings onto a notepad.
+They can be found under the :guilabel:`Configuration (SMTP only)` section. The :abbr:`SMTP (Simple
+Mail Transfer Protocol)` configuration settings include the server address, the security option
+needed (Use :abbr:`SSL (Secure Sockets Layer)`/:abbr:`TLS (Transport Layer Security)`), and the
+port number. The settings are needed to configured Mailjet in Odoo, which is covered in the
+:ref:`last section <maintain/mailjet-api/odoo-setup>`.
+
+.. note::
+   Here is `Mailjet's SMTP Configuration documentation <https://documentation.mailjet.com/hc/en-us/
+   articles/360043229473-How-can-I-configure-my-SMTP-parameters->`_ for more information.
+
+.. important::
+   Odoo blocks `port 25` on SaaS and Odoo.sh databases. :ref:`See reference here
```

Comment:
Please use "Odoo Online" and not "SaaS". Multiplying appellations confuses our readers.